revops/blog-posts.json

```diff
@@ -0,0 +1,182 @@
+    "Post Summary": "Cras sit amet velit id nulla tempus dictum sit amet eu nisi.\nMaecenas euismod sapien eu arcu convallis, vitae vestibulum ipsum maximus.\nPraesent nec orci at nulla consequat congue ut non arcu.\nLorem ipsum dolor sit amet, consectetur adipiscing elit.\nPraesent nec orci at nulla consequat congu",
+    "Main Image": "https://uploads-ssl.webflow.com/66c31735a489578abdbf26f1/66c3422c3188060700245da4_image18.jpeg",
+    "Thumbnail image": "https://uploads-ssl.webflow.com/66c31735a489578abdbf26f1/66c31798be4b2f0a05bc84c7_image16.jpeg",
+    "Color": "#017737",
+    "Tags": "pipedrive",
+    "Date": "Sun Aug 11 2024 17:13:14 GMT+0000 (Coordinated Universal Time)",
+    "CRM service": "pipedrive"
+  }
+]
```

Comment:
I love the idea of bootstrapping a CMS by exporting json files from somewhere Simon can edit them - great idea

Reply:
Yes, once we will figure out what field are needed for the json file we could potentially make it work this way.